experimental/builder/test/test_builder_utils.cpp

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
+
+#include <gtest/gtest.h>
+#include <ck/utility/sequence.hpp>
+#include <array>
+#include <type_traits>
+
+#include <ck_tile/builder/builder_utils.hpp>
+#include "testing_utils.hpp"
+
+namespace ck_tile::builder {
+namespace {
+
+class BuilderUtils : public ::testing::Test
+{
+};
+
+TEST(BuilderUtils, ToSequence)
+{
+    static constexpr std::array<int, 4> test_arr{1, 2, 8, 16};
+    using result_seq   = to_sequence_v<test_arr>;
+    using expected_seq = ck::Sequence<1, 2, 8, 16>;
+    EXPECT_TRUE((std::is_same_v<result_seq, expected_seq>));
+}
+
+TEST(BuilderUtils, ToSequenceEmpty)
+{
+    static constexpr std::array<int, 0> test_arr{};
+    using result_seq   = to_sequence_v<test_arr>;
+    using expected_seq = ck::Sequence<>;
+    EXPECT_TRUE((std::is_same_v<result_seq, expected_seq>));
+}
+
+TEST(BuilderUtils, StringLiteral)
```

Comment:
Why is this test called BuilderUtils - StringLiteral? This is not what the test is. It shouldn't go in the file test_builder_utils.hpp, either.

The only files should be testing_utils.hpp, testing_utils.cpp, and the test for them, test_testing_utils.cpp. Builder utils don't belong here.

Reply:
Actually, it looks like my might not have pushed the latest changes you have. Your comments look like they may be ahead of the code in this PR.